src/modules/roc_netio/target_uv/roc_netio/transceiver.cpp

```diff
@@ -203,6 +175,26 @@ void Transceiver::run() {
     roc_log(LogDebug, "transceiver: finishing event loop");
 }
 
+void Transceiver::close(IElement& element) {
+    roc_log(LogDebug, "transceiver: removed element: port %s",
```

Comment:
I'd change the log message to something like "asynchronous close finished".

Reply:
Done